.ado/jobs/playground.yml

```diff
@@ -84,15 +111,17 @@ jobs:
             /p:PreferredToolArchitecture=$(MSBuildPreferredToolArchitecture)
             /p:PlatformToolset=$(MSBuildPlatformToolset)
             $(BuildWinUI3Properties)
-            /p:BaseIntDir=$(BaseIntDir) 
+            /p:BaseIntDir=$(BaseIntDir)
+            /p:PackageCertificateKeyFile="packages\playground-Key.pfx"
+        condition: eq('${{ parameters.BuildEnvironment }}', 'Continuous')
```

Comment:
it would be useful to publish the Playground AppX (+symbols) from CI so we can install it locally to debug